### PR TITLE
APIv3 append offset and limit to getProfiles etag

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Appended the offset and limit to getProfiles etag so it returns results when those params are different than previous requests

### CODE REVIEW

File: api/user.go [hunk 1]
```diff
@@ -437,7 +437,7 @@ func getProfiles(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	etag := app.GetUsersEtag()
+	etag := app.GetUsersEtag() + params["offset"] + params["limit"]
```
Comment (jwilander) [inline api/user.go]: Maybe just throw a `.` between them so that offset 1 and limit 11 doesn't get confused for offset  11 and limit 1

**Comment by enahum** [comment]
@jwilander done

**Comment by coreyhulen** [comment]
This seems totally weird since the offset and limit are part of the URL path so the etag should change based on the page?

**Comment by enahum** [comment]
@coreyhulen for some weird reason is not changing with `fetch` in the RN app so that's why this PR

**Pull request merged**